src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -274,6 +274,8 @@ public class DeploymentOptions
 
         private Dictionary<string, string> _displayNameMapDict;
 
+        public Dictionary<string, int> IncludeObjectsTable;
```

Comment:
does this need to be public?

Reply:
Yes this Dictionary will be used in ADS SQLDB/SC extensions